test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetNewTemplatesTests.cs

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Xml.Linq;
+using FluentAssertions;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace Dotnet.Integration.Test
+{
+    [Collection("Dotnet Integration Tests")]
+    public class DotnetNewTemplatesTests
```

Comment:
These tests are testing pack, I think the name should be clearer about that. We already have a `PackCommandTest`, which unfortunately is already huge, making it hard to determine if equivalent tests exist, but is probably a better place for these tests, until someone stats splitting `PackCommandTest` by some kind of category.

Reply:
Ok. Moved to `PackCommandTest`. But 2 tests moved to `DotnetRestoreTests.cs`, because they don't pack due to `IsPackable` is false.